### PR TITLE
Home work

### PR DESCRIPTION
**Pull request by Ihor-Pavliuk**
hw

### CODE REVIEW

File: Home_work1.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+text_2 = 'again'
+print (text_1, text_2, sep=',', end='!\n')
+print ('{0},{1}' .format(text_1, text_2), end='!\n')
+print (f"{text_1}, {text_2}!\n\n")
+
+s = '#' # це не правильно написаний код, тому що сторонній особі нічого не зрозуміло, просто було цікаво чи можно це зробити в одній строці 
+print (f"{s}{s}{s}{s}{s}{s}{s}{s}{s}\n{s}\t{s}\n{s}\t{s}\n{s}\t{s}\n{s}{s}{s}{s}{s}{s}{s}{s}{s}\n\n{s}\t{s}\n{s}\t{s}\n{s}{s}{s}{s}{s}{s}{s}{s}{s}\n{s}\t{s}\n{s}\t{s}\n")
+# ось домашка
+print (f"{s}{s}{s}{s}{s}{s}{s}{s}{s}\n{s}\t{s}\n{s}\t{s}\n{s}\t{s}\n{s}{s}{s}{s}{s}{s}{s}{s}{s}\n")
+print (f"{s}\t{s}\n{s}\t{s}\n{s}{s}{s}{s}{s}{s}{s}{s}{s}\n{s}\t{s}\n{s}\t{s}\n")
```
Comment (UmterRick) [inline Home_work1.py:13]: String formatting usage is overhead here, just use # symbol to make code more readable. (And also you can try to use triple quotes with \n and \t if you want)

File: Home_work1.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Декілька способів, можно ще модулі підв'язати, але це буде окремий файл
+print ('Hello world')
+text_1 = 'Hello world' 
+text_2 = 'again'
+print (text_1, text_2, sep=',', end='!\n')
+print ('{0},{1}' .format(text_1, text_2), end='!\n')
+print (f"{text_1}, {text_2}!\n\n")
+
```
Comment (UmterRick) [inline Home_work1.py:8]: Complete task 2, where you put expected result in comment and create this result using features of print after comment

**Review by UmterRick — CHANGES_REQUESTED** [review]
Task 2: TO DO
Task 3: reformat code

File: Home_work1.py (hunk 1 above)
Comment (Ihor-Pavliuk) [reply Home_work1.py]: Done

File: Home_work1.py (hunk 2 above)
Comment (Ihor-Pavliuk) [reply Home_work1.py:12]: Done

**Pull request merged**